include/igl/read_triangle_mesh.cpp

```diff
@@ -43,7 +43,7 @@ IGL_INLINE bool igl::read_triangle_mesh(
     bool success = readOBJ(str,V,TC,N,F,FTC,FN);
     for(auto & v : V)
     {
-      v.resize(std::min(v.size(),(size_t)3));
+      v.resize((std::min)(v.size(),(size_t)3));
```

Comment:
Wow, what a mess. So every time we use `std::min()` we should be using `(std::min)()`?